### DISCUSSION TITLE
Zed  + serial monitor / console?

### DISCUSSION THREAD

**Discussion by vacquah** [zed-industries/zed#6612] [Feature Requests]
Does zed have an inbuilt serial console?

**Comment by jansol** [comment]
No, but you can use any of the popular serial consoles (`tio`, `picocom`, `minicom` etc) via the builtin terminal.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
